content/workshops/openshift_4_101/lab7-labels.md

```diff
@@ -68,35 +86,20 @@ Next let's look at the log for the pod running our application.
 <blockquote>
 Click the pod for the dc metro map webapp (it shoud have a status of Running)
 </blockquote>
-<img src="../images/ocp-lab-labels-poddetails.png" width="900"></br>
+<img src="../images/ocp-lab-labels-poddetails.png" width="500"></br>
 Here, at the top, you can see the labels on this pod
 
 <blockquote>
-Click the "Actions" button, then click "Edit YAML" for the pod
+Click the "Actions" button, then click "Edit Labels" for the pod
```

Comment:
If "Actions" is a drop-down, then:    Select 'Edit Labels' from the Actions drop-down menu.